roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- **Make adjustments**: Maintainers are responsible for removing inactive committers. A committer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+As long as a Committer follows our Code of Conduct, makes reasonable contributions to the project,
+and is not violating any of our policies, they will keep their Committer role.
+
+Project maintainers may remove a committer if:
+
+- Breaks Code of Conduct.
+- Shown no activity for 6 months and does not respond when contacted by the project maintainers or LFDT staff.
+- Keeps making bad decisions (their approvals and contributions are negatively affecting the project's performance).
+
+The addition or removal of committers must be decided by a project maintainer vote. The voting must be held via a GitHub PR (by proposing a change to the `config.yaml` file).
+
+### Maintainers
+
+The Maintainers of the project are the people that are responsible for the project.
+
+A Maintainer should drive the project's roadmap/vision and take care of topics like community meetings,
+the general structure of the project (fits our best practices, is secure, ...), and so on.
+
+In addition, the Maintainers should report project updates to the Hiero **Technical Steering Committee** (TSC) meeting.
+A Committer of a project can be nominated to become a Maintainer:
+
+- **Define your project's needs**: A repository should have 1 to 3 maintainers to ensure a project's sustainability and resilience. It is not necesarily a goal for the committer to become a maintainer unless the project's needs require it to have more maintainers.
+- **Nominate a maintainer**: To promote a candidate, a vote should be held in a GitHub PR (by proposing a change to the config.yaml file) and by making mention of the basis reached for nominating the Maintainer. The template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md) should be used to creating a nomination.
+- **Approve the nomination**: All maintainers should vote on that nomination and the vote passes when a majority of maintainers vote to proceed. The voting procedure is explained in the same template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md).
+- **Make adjustments**: A maintainer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+
+Maintainers are responsible for keeping track of project Issues and Discussions and help pushing inactive conversations. It is a responsibility of the maintainers to close Issues and Discussions that are considered outdated or stale to the current goals of the project.
+
+Project maintainers or the TSC may remove a maintainer who:
```

Comment:
Is that maintainer becoming a committer or a contributor?

Reply:
Not sure I understand your question. 
"Project maintainers or the TSC may remove a maintainer who" refers to Maintainers or TSC members having the ability to remove another Maintainer in case that Maintainer is not performing as expected. 

In such case, Committers won't be able to suggest removal of Maintainers. 

In other cases, mentioned in this doc, both Maintainers and Committers can suggest a Committer removal.

Reply:
Will a removed maintainer become a committer or a contributor. That was my question